src/components/NavbarShoppingList.jsx

```diff
@@ -11,6 +15,18 @@ export function NavbarShoppingList() {
 						<Nav.Link href="/">Home</Nav.Link>
 						<Nav.Link href="/list">List</Nav.Link>
 						<Nav.Link href="/add-item">Add Item</Nav.Link>
+
+						{/* This will make 'Exit List' visible ONLY when route is on /list */}
+						{location.pathname === '/list' ? (
+							<Nav.Link
+								href="/"
+								onClick={() =>
+									localStorage.removeItem('tcl-shopping-list-token')
+								}
+							>
+								Exit List
+							</Nav.Link>
+						) : null}
```

Comment:
This is all super clear! I haven't used useLocation() before, so I definitely learned something from this!

Reply:
Cool!  I'm learning even more thanks to this.  Nice work

Reply:
+1 Love seeing you how you and the rest of the team develop UI, I'm enjoying all that I'm learning from you!